### PR TITLE
Fixing compilation error that occurs when DevHelper creates a new filter

### PR DESCRIPTION
**Pull request by joeykleingers**
Fixing an error where the filter's implementation file would not generate the colon character before the initializer list.

### CODE REVIEW

File: Source/DevHelper/Template/Filter/Filter.cpp.in [hunk 1]
```diff
@@ -15,7 +15,8 @@
 // -----------------------------------------------------------------------------
 //
 // -----------------------------------------------------------------------------
-@ClassName@::@ClassName@()@InitList@
+@ClassName@::@ClassName@() : 
+@InitList@
 {
```
Comment (imikejackson) [inline Source/DevHelper/Template/Filter/Filter.cpp.in:19]: What if the InitList is empty? I think that the code itself should figure out if the InitList is empty and if it isn't empty then add the ':' character.

Comment (joeykleingers) [reply Source/DevHelper/Template/Filter/Filter.cpp.in]: InitList is never empty because by default it at least puts in AbstractFilter()

Comment (joeykleingers) [reply Source/DevHelper/Template/Filter/Filter.cpp.in]: Wait, never mind it can be empty.

**Comment by imikejackson** [comment]
Looks good. Merge it in.

**Pull request merged**